decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
```

Comment:
Consider importing this file with `require_relative` in order to allow the inheritance work. When I try to run your code, I get this error.

<img width="585" alt="error" src="https://github.com/Ellyboi/school-library/assets/84930355/d7b4771e-6e33-48cd-a2a7-9e5ab468b516">


NOTE: Ensure to execute you code before asking for review again